web/api/v2/verify/graphql/atomic-upsert-nullifier.graphql

```diff
@@ -0,0 +1,20 @@
+mutation AtomicUpsertNullifier($action_id: String!, $nullifier_hash: String!) {
```

Comment:
Here because we cannot do _inc with upsert. We insert an action with an empty value, then increment. This solves the error of duplicate constraints since if it sees a duplicate it just does nothing as update_columns is empty

Reply:
We dont rely on the uses value coming in anymore since that would fail if two proofs are submitted together as the uses comes from a previous graphql query. We should count every valid incoming request